### PR TITLE
Utku / Homework Week 1

### PR DESCRIPTION
**Pull request by u61**

### CODE REVIEW

File: Week1/homework/create-database.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  user: 'hyfuser',
+  password: '7566',
+  database: 'userdb'
+});
+
+const execQuery = util.promisify(connection.query.bind(connection));
+
+async function seedDatabase() {
+  const CREATE_DATABASE_AND_LOGIN = `
+    CREATE DATABASE IF NOT EXISTS world; USE world`;
```
Comment (zhaobn) [inline Week1/homework/create-database.js]: Or, you can take out the table creation code, put them in another place where you tell `mysql.js` to use `database: world`.

File: Week1/homework/create-database.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  const CREATE_COUNTRY_TABLE = `
+    CREATE TABLE IF NOT EXISTS country (
+      Name TEXT,
+      Population INT,
+      Surface_Area_km2 INT,
+      Continent TEXT
+    );`;
+
+  const CREATE_CITY_TABLE = `
+    CREATE TABLE IF NOT EXISTS city (
```
Comment (zhaobn) [inline Week1/homework/create-database.js]: This schema is perhaps enough for the homework, but a better practice is to also give each row an ID. We will talk about in next lessons, pay attention to "key"s.

File: Week1/homework/exec_query.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+const select_query = [
+  `SELECT name FROM country WHERE population >= 8000000;`,
+  `SELECT name FROM country WHERE name LIKE '%land%'`,
+  `SELECT name, population FROM city WHERE population BETWEEN 500000 AND 1000000`,
+  `SELECT name FROM country WHERE continent LIKE '%Europe%'`,
+  `SELECT name FROM country ORDER BY surface_area_km2 DESC`
+];
+
+for (let i of select_query) {
```
Comment (zhaobn) [inline Week1/homework/exec_query.js]: Extra challenge: can you make this function to accept an input from the user, say, a number, and then give the requested answer? For example, if I type 2, I'm expecting the second question from the homework.

File: Week1/homework/create-database.js (hunk 1 above)
Comment (u61) [reply Week1/homework/create-database.js]: But I especially used this syntax to put them together.
Is there anything wrong with it ?

File: Week1/homework/create-database.js (hunk 2 above)
Comment (u61) [reply Week1/homework/create-database.js]: IDs are added.

**Pull request closed without merging**